subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/StartParamsValidatingActionExecuter.java

```diff
@@ -60,6 +62,16 @@ public Object execute(BuildAction action, BuildRequestContext requestContext, Bu
                 throw new IllegalArgumentException(String.format("The specified settings file '%s' is not a file.", startParameter.getSettingsFile()));
             }
         }
+        if (startParameter.getInitScripts().size() > 0) {
```

Comment:
This `if` is unnecessary.

Reply:
Agreed, removed in 2a249918cf7e3bf430563cca65b908c3b5305eb1